### ISSUE TITLE
Lesser Felguard 3772 doesn't fight back.

### ISSUE THREAD

**Issue by StraysFromPath** [ZhengPeiRu21/mod-individual-progression#229]
![image](https://github.com/ZhengPeiRu21/mod-individual-progression/assets/119371540/10626037-b558-4215-a6ee-aa2d622a6df9)

**Comment by StraysFromPath** [comment]
![image](https://github.com/ZhengPeiRu21/mod-individual-progression/assets/119371540/b6866a68-73f5-424b-978c-4462bd9ba539)

3924 also doesn't fight back.

This has to be some kind of wider issue involving casters/npc ai?

**Comment by Grimfeather** [comment]
I tested this. They fight back normally for me.
There may be something wrong with your server.

https://i.gyazo.com/5a608c05ccf757b8e9bb87b3570f2bc0.jpg

3924 is the entry number for Thistlefur Shaman. That's the 'creature_template' number.
So that's all thistlefur shamans.

If you meant one specific thistlefur shaman you will have to give me the guid number from the 'creature' table.

**Comment by StraysFromPath** [comment]
> I tested this. They fight back normally for me. There may be something wrong with your server.
> 
> https://i.gyazo.com/73f9bea15683c61db985818d38f6cffa.jpg
> 
> 3924 is the entry number for Thistlefur Shaman. That's the 'creature_template' number. So that's all thistlefur shamans.
> 
> If you meant one specific thistlefur shaman you will have to give me the guid number from the 'creature' table.
> 
> Lesser felguard fights back normally as well. https://i.gyazo.com/459bde468aafa87da22c12af9c5f8598.jpg

Can you try a few others? Upon further testing on my end it only seems to happen sometimes/with some of them.

**Comment by Grimfeather** [comment]
I killed all the shamans and lesser felguards in Ashenvale I could find. they all fought back.

Tested with AzerothCore version I compiled just a week ago.
AzerothCore rev. 46062dd070a8 2024-01-20 20:35:59 +0700 (npcbots_3.3.5 branch) (Win64, Release, Static) 
modules: npcbots, individual progresssion, lua engine and reagent bank 

You using the module preview build?
You having this problem with all classes?

Give me some more specific information so I can try to reproduce it.
Because right now I can't.

Are they evading your attacks? Do they move at all? Are the felguards self buffing still?
maybe there is something wrong with your maps and the enemies can't reach you.

This does not look like it's related to Individual progression.

**Comment by StraysFromPath** [comment]
> I killed all the shamans and lesser felguards in Ashenvale I could find. they all fought back.
> 
> Tested with AzerothCore version I compiled just a week ago. AzerothCore rev. 46062dd070a8 2024-01-20 20:35:59 +0700 (npcbots_3.3.5 branch) (Win64, Release, Static) modules: npcbots, individual progresssion, lua engine and reagent bank
> 
> You using the module preview build? You having this problem with all classes?
> 
> Give me some more specific information so I can try to reproduce it. Because right now I can't.
> 
> Are they evading your attacks? Do they move at all? Are the felguards self buffing still? maybe there is something wrong with your maps and the enemies can't reach you.
> 
> This does not look like it's related to Individual progression.

I reinstalled my maps, but they're only the exact same files which came with the prebuilt before, and nothing changed.

I am on the prebuild so it HAS to be something on the module or the prebuild build end.

Felguards are self buffing. They move/patrol as normal but if you hit them they just ignore your existence entirely.

They do not attack, they do not turn, they do not evade, you can kill them.

This has been reported on 27 maps ranging from barrens to ashenvale to elwynn to searing gorge.

Same issue here with a player and swiftmane who runs really fast around barrens and he killed it by standing on its path and whacking it once every 2-3 mins as it made its loop:

![image](https://github.com/ZhengPeiRu21/mod-individual-progression/assets/119371540/73a70525-1530-43bd-b352-48fbcc489067)

**Comment by Grimfeather** [comment]
I don't know what to tell you.
I just installed the preview build and tested the issues you're having again.
I did not change any settings at all. Just downloaded the files, copy, paste, overwrite, done. play
It all works just fine.

https://github.com/ZhengPeiRu21/mod-individual-progression/releases/tag/20240122
Make 100% sure you follow those instructions.

Installation Instructions (If not yet installed)

Extract ModIndividualProgressionPreview to any folder (7-zip is required to extract the 7z archive)
Download latest pre-extracted client map data from https://github.com/wowgaming/client-data/releases
Extract contents of client map data into the "data" directory
Download IndividualProgression [dbc.zip](https://github.com/ZhengPeiRu21/mod-individual-progression/raw/master/optional/dbc.zip)
Place the .dbc files from dbc.zip into the data/dbc directory, overwriting existing files
Place the .mpq files from dbc.zip into the Data directory of your WoW client
Double-click play.bat to start the server - the MySQL database, AuthServer and WorldServer will all start
Remember to change the realmlist.wtf in your client to point to your local server (127.0.0.1).
It is not necessary to install the update when installing from this release - it is already included in the install

-------------

I don't know what else I can do to help. You must be making a mistake somewhere during the installation.
I can't reproduce your issues with the preview build of this module. with the default settings.
